model/ivector/online_cmvn.conf

```diff
@@ -0,0 +1 @@
+# configuration file for apply-cmvn-online, used in the script ../local/run_online_decoding.sh
```

Comment:
Don't forget to move this into the `SpeechRecognition` data director. Do we need all of the files inside `model`, can we take just a subset?